### PR TITLE
Major design changes

### PR DESCRIPTION
**Pull request by viliusmac**
Login, register, footer and header changed

### CODE REVIEW

File: SmartSaver.MVC/Views/Authentication/Login.cshtml [hunk 1]
```diff
@@ -9,29 +9,57 @@
 </head>
 
 <br />
-<h1>User Login</h1>
 <br />
```
Comment (saltanovas) [inline SmartSaver.MVC/Views/Authentication/Login.cshtml]: I see that you use `<br>` everywhere in your code. Don't know which one is better, but use `<br>` in here as well

File: SmartSaver.MVC/Views/Authentication/Login.cshtml [hunk 2]
```diff
@@ -9,29 +9,57 @@
...
+                    <div class="row" style="padding-top:5px;">
+                        <div class="col-md-12">
+                            <button type="submit" name="provider" value="Facebook" class="btn btn-lg btn-google btn-block text-uppercase btn-outline">
+                                Facebook Login⠀
+                                <img src="https://icons.iconarchive.com/icons/danleech/simple/16/facebook-icon.png" style="background-size: 5px;">
+                            </button>
+                        </div> <br>
+                    </div>
+                </form>
+
```
Comment (saltanovas) [inline SmartSaver.MVC/Views/Authentication/Login.cshtml]: as you didn't add additional new line above `<form>`, these new lines are redundant. Or add new line there.

File: SmartSaver.MVC/Views/Authentication/Register.cshtml [hunk 3]
```diff
@@ -7,47 +7,64 @@
     <link rel="stylesheet" type="text/css" href="~/css/generalDesign.css" />
     <title>@ViewBag.Title</title>
 </head>
+<br>
+<br>
+<body onload="UsernameCheck(); EmailCheck()" style="background-size:cover; background-image: url(https://images.unsplash.com/photo-1475650522725-015d35677789?ixlib=rb-1.2.1&ixid=MXwxMjA3fDB8MHxwaG90by1wYWdlfHx8fGVufDB8fHw%3D&auto=format&fit=crop&w=1050&q=80)">
+    <div class="card center" style="max-width: 500px; padding: 20px 20px 20px 20px;">
+        <div class="card-block">
+            <div class="col-md-12">
+                <form method="post">
```
Comment (saltanovas) [inline SmartSaver.MVC/Views/Authentication/Register.cshtml]: `asp-action`, `asp-controller` and `asp-route-returnUrl` is missing in the `<form>`. But this is my bad. Will change that

File: SmartSaver.MVC/Views/Shared/_Layout.cshtml [hunk 4]
```diff
@@ -11,10 +11,10 @@
 </head>
 <body>
 <header>
-    <nav class="navbar navbar-expand-sm navbar-light" style="background-color: rgb(177, 226, 213)">
+    <nav class="navbar navbar-expand-sm navbar-light" style="background-color: rgb(30, 51, 45)">
```
Comment (saltanovas) [inline SmartSaver.MVC/Views/Shared/_Layout.cshtml:14]: can this style be moved to .css file?

File: SmartSaver.MVC/Views/Shared/_Layout.cshtml [hunk 5]
```diff
@@ -58,8 +58,8 @@
         </main>
     </div>
 
-    <footer class="border-top footer text-muted">
-        <div class="container">
+    <footer class="border-top footer text-muted" style="background-color: #0e1714;">
+        <div class="container" style="color:white;">
```
Comment (saltanovas) [inline SmartSaver.MVC/Views/Shared/_Layout.cshtml:62]: again I believe the style can be moved to .css file?

File: SmartSaver.MVC/wwwroot/css/generalDesign.css [hunk 6]
```diff
@@ -49,4 +28,125 @@ input[type="date"]:valid::before {
...
+
+#infoMessage p {
+    color: red !important
+}
+
+.btn-google {
+    color: #545454;
+    background-color: #ffffff;
+    box-shadow: 0 1px 2px 1px #ddd
+}
```
Comment (saltanovas) [inline SmartSaver.MVC/wwwroot/css/generalDesign.css]: this one can have more meaningful name as you use this button for all external authentication providers

**Review by saltanovas — COMMENTED** [review]
Quite impressive job. Keep it up!

File: SmartSaver.MVC/Views/Authentication/Login.cshtml [hunk 7]
```diff
@@ -9,29 +9,57 @@
...
+                    <div class="line-separator"></div>
+                </div>
+                <form method="post" asp-action="ExternalLogin" asp-controller="Authentication" asp-route-returnUrl="@Context.Request.Query["returnurl"]">
+                    <div class="row">
+                        <div class="col-md-12">
+                            <button type="submit" name="provider" value="Google" class="btn btn-lg btn-google btn-block text-uppercase btn-outline">
+                                Google Login⠀
+                                <img src="https://img.icons8.com/color/16/000000/google-logo.png">
+                            </button>
+                        </div> <br>
```
Comment (povilasleka) [inline SmartSaver.MVC/Views/Authentication/Login.cshtml]: 'br' element is unnecessary here, it does not do anything. However, if you are trying to achieve more space between elements with 'br', you should better use mb-1 , mb-2, ... , mt-1, mt-2 ... bootstrap classes.

File: SmartSaver.MVC/Views/Authentication/Login.cshtml [hunk 8]
```diff
@@ -9,29 +9,57 @@
...
+                    </div>
+                    <div class="row" style="padding-top:5px;">
+                        <div class="col-md-12">
+                            <button type="submit" name="provider" value="Facebook" class="btn btn-lg btn-google btn-block text-uppercase btn-outline">
+                                Facebook Login⠀
+                                <img src="https://icons.iconarchive.com/icons/danleech/simple/16/facebook-icon.png" style="background-size: 5px;">
+                            </button>
+                        </div> <br>
```
Comment (povilasleka) [inline SmartSaver.MVC/Views/Authentication/Login.cshtml]: 'br' element is unnecessary here, it does not do anything. However, if you are trying to achieve more space between elements with 'br', you should better use mb-1 , mb-2, ... , mt-1, mt-2 ... bootstrap classes.

**Comment by saltanovas** [comment]
Made some slight changes with our design, resolved conversations.

**Pull request merged**